### ISSUE TITLE
Unable to use Loftee plugin from grch38 branch

### ISSUE THREAD

**Issue by parunachalam1** [konradjk/loftee#58]
We have downloaded loftee plugin from grch38 branch and tried to use it but encountered the below error. Could you please have a look and assist ?

_WARNING: Failed to compile plugin LoF: Attempt to reload Bio/DB/BigFile.pm aborted.
Compilation failed in require at /path/to/vep/96/ensembl-vep/Bio/DB/BigWig.pm line 7.
BEGIN failed--compilation aborted at /path/to/vep/96/ensembl-vep/Bio/DB/BigWig.pm line 7.
Compilation failed in require at /path/to/vep/plugin/loftee-grch38/loftee/gerp_dist.pl line 2.
BEGIN failed--compilation aborted at /path/to/vep/plugin/loftee-grch38/loftee/gerp_dist.pl line 2.
Compilation failed in require at /path/to/vep/plugin/loftee-grch38/loftee/LoF.pm line 27.
Compilation failed in require at (eval 72) line 2.
BEGIN failed--compilation aborted at (eval 72) line 2._

I downloaded the BigWig.pm and BigFile.pm from https://metacpan.org/release/Bio-BigFile .

Below is the command that I have executed, 
`vep -I /tmp/Loftee/loftee_chr1Vars_b38 -o test1_b38_loftee_out --offline --cache --dir_cache /path/to/vep/96/ensembl-vep/.vep --assembly GRCh38 --cache_version 93 --vcf --force_overwrite --plugin LoF,loftee_path:/path/to/vep/plugin/loftee-grch38/loftee,human_ancestor_fa:/tmp/vep/Build-38/human_ancestor.fa.gz,gerp_bigwig:/tmp/vep/Build-38/gerp_conservation_scores.homo_sapiens.GRCh38.bw,gerp_database:/tmp/vep/Build-38/gerp_conservation_scores.homo_sapiens.GRCh38.bw,conservation_file:/tmp/phylocsf_gerp.sql --dir_plugins /path/to/vep/plugin/loftee-grch38/loftee`

**Comment by AniketMishra** [comment]
I am experiencing the same problem. May I ask how you resolved this issue?

**Comment by BGI-hys** [comment]
I am experiencing the same problem. May I ask how you resolved this issue?

**Comment by boboppie** [comment]
@AniketMishra @BGI-hys 
I had the same problem, the root is the new versions of jksrc.zip results in a failure to load the tbx functions (see https://github.com/GMOD/GBrowse-Adaptors/issues/17). So I download v334 (see https://github.com/ucscGenomeBrowser/kent/releases/tag/v334_base) and compiled and installed it following BigFile readme (http://www.cpan.org/modules/by-module/Bio/Bio-BigFile-1.07.readme).

**Comment by kulmsc** [comment]
If the above solution did not work please see #51.  In short, the instructions at https://useast.ensembl.org/info/docs/tools/vep/script/vep_download.html#bigfile work well.

**Comment by sahwa** [comment]
I had this error and solved it by doing the following:


```
#This checks what the syntactic or compilation error is with the module - just change the path to whatever the offending .pm file 
perl -c /users/ckb/aey472/cpanm/lib/perl5/x86_64-linux-thread-multi/Bio/DB/BigWig.pm
```

This returned:

>Can't load '/users/ckb/aey472/cpanm/lib/perl5/x86_64-linux-thread-multi/auto/Bio/DB/BigFile/BigFile.so' for module Bio::DB::BigFile: libssl.so.1.1: cannot open shared object file: No such file or directory at /apps/eb/2020b/skylake/software/Perl/5.32.0-GCCcore-10.2.0/lib/perl5/5.32.0/x86_64-linux-thread-multi/DynaLoader.pm line 193.
 at /users/ckb/aey472/cpanm/lib/perl5/x86_64-linux-thread-multi/Bio/DB/BigWig.pm line 7.
Compilation failed in require at /users/ckb/aey472/cpanm/lib/perl5/x86_64-linux-thread-multi/Bio/DB/BigWig.pm line 7.
BEGIN failed--compilation aborted at /users/ckb/aey472/cpanm/lib/perl5/x86_64-linux-thread-multi/Bio/DB/BigWig.pm line 7.

So basically it couldn't find the openssl.so.1.1 shared object. I solved this by doing

```
module load OpenSSL/1.1
```

Or any other way to make sure it know where the shared object is. Hope this helps!
